src/main/java/kr/co/onehunnit/onhunnit/config/WebSocketHandler.java

```diff
@@ -42,6 +58,13 @@ protected void handleTextMessage(WebSocketSession session, TextMessage message)
 		locationRepository.save(location);
 	}
 
+	boolean isNotContainAllField(JSONObject jsonObject) {
+		if (jsonObject.containsKey("patientId") && jsonObject.containsKey("longitude") && jsonObject.containsKey("latitude")) {
+			return false;
+		}
+		return true;
+	}
```

Comment:
이런 함수들은 주의할게. 부정적인? 함수면 헷갈릴 가능성이 커요. 그래서 isContainAllField가 좋을 것 같아요.
그리고 좀더 클린하게
```suggestion
boolean isContainAllField(JSONObject jsonObject) {
    return jsonObject.containsKey("patientId") && jsonObject.containsKey("longitude") && jsonObject.containsKey("latitude");
}
```